src/components/MasterDetails.js

```diff
@@ -222,7 +229,7 @@ export default class MasterDetails extends React.Component<MasterDetailsDefaultP
             </div>
           </Col>
           <Col lg={detailsWidth} md={detailsWidth} sm={12} xs={12} style={{ paddingLeft: halfPadding, paddingRight: 0 }}>
-            <Detail data={detailsRow} />
+            <Detail data={detailsRow} {...this.props.detailsProps} />
```

Comment:
Note that if the detailsProps contains a "data" member, it will override the one you're explicitly passing here. (In JSX, the spread works the same as in JavaScript itself where items to the right override things that precede them.) I think you can just change the order, or you can use the Object.assign() method and then spread the result...